mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -54,27 +68,48 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 	METHOD method_1013 isValid ()Z
 	METHOD method_1014 expand (D)Lnet/minecraft/class_238;
 		ARG 1 value
-	METHOD method_17939 getXSize ()D
-	METHOD method_17940 getYSize ()D
-	METHOD method_17941 getZSize ()D
+	METHOD method_17939 getXLength ()D
+	METHOD method_17940 getYLength ()D
+	METHOD method_17941 getZLength ()D
 	METHOD method_18804 stretch (Lnet/minecraft/class_243;)Lnet/minecraft/class_238;
+		ARG 1 scale
 	METHOD method_19316 from (Lnet/minecraft/class_3341;)Lnet/minecraft/class_238;
+		ARG 0 mutable
 	METHOD method_989 offset (DDD)Lnet/minecraft/class_238;
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 	METHOD method_990 getMax (Lnet/minecraft/class_2350$class_2351;)D
 		ARG 1 axis
 	METHOD method_991 union (Lnet/minecraft/class_238;)Lnet/minecraft/class_238;
+		ARG 1 box
 	METHOD method_992 rayTrace (Lnet/minecraft/class_243;Lnet/minecraft/class_243;)Ljava/util/Optional;
+		ARG 1 min
+		ARG 2 max
 	METHOD method_993 intersects (Lnet/minecraft/class_243;Lnet/minecraft/class_243;)Z
+		ARG 1 from
+		ARG 2 to
 	METHOD method_994 intersects (Lnet/minecraft/class_238;)Z
 		ARG 1 box
-	METHOD method_995 averageDimension ()D
+	METHOD method_995 getAverageSideLength ()D
```

Comment:
Why change this? "Dimension" means "side length" but is more concise.

> [a measurable extent of some kind, such as **length, breadth, depth, or height**](https://www.google.com/search?q=dimension)

Reply:
I think "side length" is clearer, but that's really a matter of taste.

Reply:
I'm with Pyro on this one. Not only that, but `dimension` is very similar to `world dimension`, `entity dimensions`, and just the term itself implies this method returns a vector, or therwise multiple scalars to describe the "dimensions" of the box. (see: `BlockBox.getDimensions`)

`Length` on the other hand, is always a scalar.